src/gui/shell.cpp

```diff
@@ -969,6 +968,25 @@ void Shell::handleLineSpace(const QVariant& value) noexcept
 	resizeNeovim(size());
 }
 
+void Shell::handleCloseEvent(const QVariantList &args) noexcept
```

Comment:
nitpick: `QVariantList& args`

Sorry for the nitpicks. Only mentioned since we're changing other code anyways :)